modules/data.atmosphere/R/upscale_met.R

```diff
@@ -1,87 +1,85 @@
-substrRight <- function(x, n) {
-  substr(x, nchar(x) - n + 1, nchar(x))
-}
 
 ##' upscale_met upscales the temporal resolution of a dataset
 ##' @name upscale_met
 ##' @title upscale_met
 ##' @export
-##' @param outfolder
-##' @param input_met - met dataset
-##' @param site.id
-##' @param reso - numeric value in HOURS that is the desired resolution for the dataset to be upscaled to
-##'               # might need to change this for monthly/yearly option later
-##' @param overwrite
-##' @param verbose
-##' @author James Simkins
+##' @param input_met path to netcdf file containing met dataset
+##' @param outfolder path to directory where output should be saved
+##'           Output is netcdf with named as <input_met_filename>.upscaled.nc
+##' @param resolution numeric value that is the desired resolution for the dataset to be upscaled to
```

Comment:
6 hours was just me leaving the previous default. 

I'll defer to others on whether units should be configurable, but have been assuming the unit check is necessary anyway -- If the upstream met2CF conversions be trusted to always produce days, this code could be simplified.